src/toxext.h

```diff
@@ -53,6 +53,12 @@ void toxext_iterate(struct ToxExt *toxext);
 /**
  * Callback for your extension. When the toxext library receives a message from
  * a friend with the same uuid as was registered, this callback will be called
+ *
+ *  @param friend_id The Tox defined session friend ID of the sender.
```

Comment:
Seems like we should flag what extension is as well, but thanks for filling this in